### PR TITLE
App styling

### LINKED ISSUE #36: Add Web application styling

### PR DESCRIPTION
**Pull request by AronBuzogany**
This PR resolved #36 by adding some extra styling and making text color configurable in the config file.

### CODE REVIEW

**Review by pheyvaer — CHANGES_REQUESTED** [review]
image

- Can we add extra space between the border of the page and the logo at the top?
- Can we remove the shadows of the grey boxes?

**Comment by AronBuzogany** [comment]
> image
> 
>     * Can we add extra space between the border of the page and the logo at the top?
> 
>     * Can we remove the shadows of the grey boxes?

As of [commit 3500732](https://github.com/SolidLabResearch/generic-data-viewer/pull/41/commits/350073282900b7586fefdec2a5c4c5b5b1b0b9f6) both are resolved.

**Review by pheyvaer — CHANGES_REQUESTED** [review]
- Write "IDP" full out.
- Change font to Helvetica for accessibility.

**Comment by AronBuzogany** [comment]
> * Write "IDP" full out.
> 
>     * Change font to Helvetica for accessibility.

As of [commit e895ece](https://github.com/SolidLabResearch/generic-data-viewer/pull/41/commits/e895ecece505c6805d4fa396ea3bdeebb3e229a8) both are resolved

**Review by pheyvaer — CHANGES_REQUESTED** [review]
Font is not changed for all the text. For example, the title of the app still has a different font. The fallback should not be `serif` but `sans-serif`.

**Comment by AronBuzogany** [comment]
As of [commit f78f274](https://github.com/SolidLabResearch/generic-data-viewer/pull/41/commits/f78f2743a5e69eafad0047f20c248d758bbd815e) both are resolved

**Review by pheyvaer — CHANGES_REQUESTED** [review]

File: src/index.css [hunk 1]
```diff
@@ -1,23 +1,18 @@
 body {
   margin: 0;
   padding: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
+  font-family: "Helvetica Neue", Helvetica, Arial, serif;
```
[the remark this answers is not in the document]
Comment (AronBuzogany) [reply src/index.css]: Sorry for the confusion, I have reverted the [commit](https://github.com/SolidLabResearch/generic-data-viewer/pull/41/commits/8a295c6a7160caa7607622aada7a73383c5324db). 

You said 

> The fallback should not be serif but sans-serif

But there was no fallback `serif`

**Review by pheyvaer — APPROVED** [review]

**Pull request merged**